polaris-service/src/main/java/org/apache/polaris/service/TimedApplicationEventListener.java

```diff
@@ -38,13 +40,21 @@
 @Provider
 public class TimedApplicationEventListener implements ApplicationEventListener {
 
+  public static final String METRIC_NAME = "polaris.TimedApi";
```

Comment:
Does this get prefixed to the existing metric name? E.g., currently we publish `polaris.CatalogApi.commitTransaction`. Will it now be `polaris.TimedApi.polaris.CatalogApi.commitTransaction`?

Reply:
We still emit the former metric and a new one that's just called `polaris.TimedApi` for all API endpoints.

Reply:
Can we just make this `polaris.api`? I propose we go with the naming convention of all lowercase and underscore separated - I feel that's a much more common metric naming convention than the CamelCase we use here. I think we can deprecate the old metric names and eventually get rid of them.

Reply:
Updated!